src/dymaptic.GeoBlazor.Core/Components/Symbols/SimpleLineSymbol.cs

```diff
@@ -59,7 +59,8 @@ internal override SymbolSerializationRecord ToSerializationRecord()
     {
         return new SymbolSerializationRecord(Type, Color)
         {
-            Width = Width, LineStyle = LineStyle?.ToString().ToKebabCase()
+            Width = Width, 
+            Style = LineStyle?.ToString().ToKebabCase()
```

Comment:
As as above in the `SimpleFillSymbol` and the `FillStyle = FillStyle....` Initially, I read this as it should this change as unnecessary and keep `LineStyle = LineStyle....` to match the parameter above.  However, found the property is 'style' in the docs, so I think that answers my questions.

Reply:
One other thing.  Are we missing a couple properties on the the simpleLineSymbol?  Cap, miterLimit, and marker?

Reply:
I'm sure we are, but that is a separate issue. I will create one.